### DISCUSSION TITLE
Engine hours support

### DISCUSSION THREAD

**Discussion by augmentedagent** [hargata/lubelog#384] [General]
I see there is a checkbox for using engine hours rather than odometer, but it doesn't seem to change anything. I'm still required to enter odometer reading and there doesn't seem to be a way to set reminders based on engine hours.
Am I missing something or is this something still in the works?

**Comment by hargata** [comment]
Checking the engine hours switches the distance unit for the odometer from miles/Km to hours, so your fuel mileage for example will show in h/g or g/100h.

**Reply by litebright** [reply]
Cannot find the switch to turn off h/g. Where is it?

**Reply by hargata** [reply]
@litebright vehicle details > Use Engine Hours

<img width="410" height="105" alt="image" src="https://github.com/user-attachments/assets/82333b1d-922d-4b8f-b973-5a62ba29f114" />

**Reply by litebright** [reply]
Ah thank you!

**Comment by hawkeye426** [comment]
Is it possible to alter this to allow for partial hours, eg 102.6 hours. Right now this appears to be an integer value only.
